packages/patternfly-4/react-core/src/components/Select/CheckboxSelect.js

```diff
@@ -17,6 +17,10 @@ const propTypes = {
   isGrouped: PropTypes.bool,
   /** Currently checked options */
   checked: PropTypes.arrayOf(PropTypes.string),
+  /** Aria label for fieldset */
+  'aria-label': PropTypes.string,
+  /** Aria labelled by for fieldset */
+  'aria-labelledby': PropTypes.string,
```

Comment:
The old prop `ariaLabelledBy` is still being used in the example.

Reply:
Oh, and it looks like that's why `arialabelledby` might be getting defined on `fieldset.pf-c-form__fieldset` which shouldn't be there.

Reply:
You're killing it! Fixed in latest commit.